features/support/paths.rb

```diff
@@ -17,26 +17,31 @@ def path_to(page_name)
       '/'
 
     when /^the new vendor page$/
-      new_vendor_path
+      new_admin_vendor_path
 
     when /^the vendors page$/
-      vendors_path
+      admin_vendors_path
 
     when /^the edit vendor page$/
-      edit_vendor_path(1)
+      edit_admin_vendor_path(1)
 
     when /^the new product page$/
-      new_product_path
+      new_admin_product_path
 
     when /^the products page$/
-      products_path
+      admin_products_path
 
     when /^the edit product page$/
-      edit_product_path(1)
+      edit_admin_product_path(1)
 
     when /^the Discovery page$/
       discovery_path
 
+    when /^the products index page$/
```

Comment:
Same thing here. This should probably be "customer products page" or "customer-facing products page".